### ISSUE TITLE
Memory 000 in the playback list.

### ISSUE THREAD

**Issue by jimandreas** [jimandreas/ReplicaIslandKotlin#14]
Fix Memory 000 in the list - see snip:
[Memory000.PNG](https://github.com/jimandreas/ReplicaIslandKotlin/blob/master/docs/img/Bug01-Memory000.PNG)

**Comment by jimandreas** [comment]
If you are here and are following along, this is a hint about debugging the game.   See AndouKun.kt:

        // If the version is a negative number, debug features (logging and a debug menu)
        // are enabled.
        const val VERSION = 14

**Comment by jimandreas** [comment]
A followup:   Chris notes in his level data analysis that:

[_"Note that Replica Island levels do not play out in chronological order, so the levels in this list are sorted in the order that they occur in the game. "_](http://replicaisland.net/index.php?view=en/player_metrics.php)

Turns out once you finish the game all levels are unlocked for revisiting.   My guess is that this was in reponse to numerous requests to replay a level - particularly the last level so you can figure out all three endings.

Also the levels "24.3 and 24.7" are probably added after Chris analyzed the player data.   I don't see them in his metrics posted to replicaisland.net.

So in short, not a bug.   This is further explained here in the [tvtropes](https://tvtropes.org/pmwiki/pmwiki.php/VideoGame/ReplicaIsland) posting:  _**"Hopeless Boss Fight: The game starts In Medias Res with Wanda helping to restore Android just after it had been defeated by Kyle. Later in the game (earlier chronologically) you play this fight, and obviously there is no way you can win it."**_

[Memory000-CLOSED](https://github.com/jimandreas/ReplicaIslandKotlin/blob/master/docs/img/Bug01-Memory000-CLOSED.PNG)
